framework/src/Volo.Abp.ExceptionHandling/Volo/Abp/ExceptionHandling/Localization/en.json

```diff
@@ -5,22 +5,25 @@
     "ValidationErrorMessage": "Your request is not valid!",
     "ValidationNarrativeErrorMessageTitle": "The following errors were detected during validation.",
     "DefaultErrorMessage": "An error has occurred!",
-    "DefaultErrorMessageDetail": "Error detail not sent by server.",
+    "DefaultErrorMessageDetail": "Error detail not sent by the server.",
     "DefaultErrorMessage401": "You are not authenticated!",
     "DefaultErrorMessage401Detail": "You should sign in to perform this operation.",
     "DefaultErrorMessage403": "You are not authorized!",
     "DefaultErrorMessage403Detail": "You are not allowed to perform this operation!",
     "DefaultErrorMessage404": "Resource not found!",
     "DefaultErrorMessage404Detail": "The resource requested could not be found on the server!",
     "EntityNotFoundErrorMessage": "There is no entity {0} with id = {1}!",
-    "AbpDbConcurrencyErrorMessage": "The data you have submitted has already changed by another user/client. Please discard the changes you've done and try from the beginning.",
+    "AbpDbConcurrencyErrorMessage": "The data you have submitted has already been changed by another user. Discard your changes and try again.",
     "Error": "Error",
     "UnhandledException": "Unhandled exception!",
     "401Message": "Unauthorized",
     "403Message": "Forbidden",
     "404Message": "Page not found",
     "500Message": "Internal Server Error",
     "403MessageDetail": "You are not authorized to perform this operation!",
-    "404MessageDetail": "Sorry, there's nothing at this address."
+    "404MessageDetail": "Sorry, there's nothing at this address.",
+    "Unauthorized": "Unauthorized",
+    "invalid_token": "Invalid token",
+    "The token is no longer valid because the user's session expired.": "The token is no longer valid because the user's session expired."
```

Comment:
![image](https://github.com/abpframework/abp/assets/9526587/f36b86ee-85a1-4844-a76a-a52cf803d314)

Reply:
The session expired message should be user-friendly:

Turkish :  `Uzun süredir işlem yapmadığınız için oturumunuz sonlanmıştır! Sayfayı yenileyin veya tekrar giriş yapın.`
English: `Your session has expired due to being idle for a long time! Refresh the page or login again.`

Reply:
In most cases, the administrator revokes the user's session.
So the message is should not be `idle for a long time`